src/build/arg/mod.rs

```diff
@@ -2702,10 +2725,12 @@ impl<'help> Arg<'help> {
     /// [`Arg::default_value_if`]: ./struct.Arg.html#method.default_value_if
     pub fn default_value_ifs<T: Key>(
         mut self,
-        ifs: &[(T, Option<&'help str>, &'help str)],
+        ifs: &[(T, Option<&'help str>, impl Into<Option<&'help str>> + Copy)],
```

Comment:
Why is there a `Copy` here?

Reply:
The slice returns a reference to the value but to convert between types you need the value, so you need to make a copy.